src/alf/atoms.ts

```diff
@@ -67,6 +67,9 @@ export const atoms = {
     zIndex: 50,
   },
 
+  overflow_visible: {
+    overflow: 'visible',
+  },
```

Comment:
Nit: Are we missing a new line here?

Reply:
I was following convention, as the others are grouped similarly without new lines.